modules/ROOT/pages/heartbeat-vm.adoc

```diff
@@ -0,0 +1,41 @@
+= Heartbeat on VM/Server
+
+This document describes how to install Pulsar Heartbeat on a virtual machine (VM) or server. For installation with the Docker image, see xref:quickstart-helm-installs.adoc[Helm Chart Installation].
+
+== Install Heartbeat Binary
+
+The release page is at https://github.com/datastax/pulsar-heartbeat/releases. The latest version is 1.0.12, which matches with the Git release tag and Docker image.
```

Comment:
Can we remove the explicit version number so that we don't have to keep this updated?